hugr/src/extension/prelude.rs

```diff
@@ -54,10 +54,20 @@ lazy_static! {
                 TypeDefBound::Explicit(crate::types::TypeBound::Eq),
             )
             .unwrap();
-
-
-        prelude
-            .add_type(
+        prelude.add_type(
+                SmolStr::new_inline("string"),
```

Comment:
I'll change the first, but inclined to keep the second "string" as it is a description rather than a name (so could in principle be different, though "string" seems an adequate description).